core/templates/dev/head/pages/question_player/question_player_directive.html

```diff
@@ -19,6 +24,13 @@ <h3>Question <[currentQuestion]>/<[totalQuestions]></h3>
         background: #FFF;background: #FFFFFF;
         height: 100px;
     }
+    .oppia-question-player-body {
+        margin: 34px 20px 0;
```

Comment:
I think we use two spaces for indentation, can you please check! If yes, then can you also resolve the indentation for the other styles in this file.

@oppia/dev-workflow-team, do we have any test for this check?

Reply:
Done

Reply:
Hi @DubeySandeep Sorry for delayed response.
We have an [issue](https://github.com/oppia/oppia/issues/6688) tracking this already.
Thanks!